randovania/game_description/pretty_print.py

```diff
@@ -94,6 +99,9 @@ def pretty_format_requirement(
 
     elif isinstance(requirement, RequirementTemplate):
         yield level, get_template_name(db, requirement)
+
+    elif isinstance(requirement, NodeRequirement):
+        yield level, requirement.node_identifier.as_string
```

Comment:
IMO displaying just the node identifier is a tad confusing:
Maybe have `Collected` before it?